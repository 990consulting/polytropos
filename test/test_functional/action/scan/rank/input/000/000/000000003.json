{
  "immutable": {
    "integer_source": 2,
    "currency_source": 2,
    "decimal_source": 0.2
  }
}