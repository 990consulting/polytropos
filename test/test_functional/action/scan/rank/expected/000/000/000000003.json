{
  "immutable": {
    "integer_source": 2,
    "currency_source": 2,
    "decimal_source": 0.2,
    "rank": 1
  }
}